README.md

```diff
@@ -31,56 +31,164 @@
 </div>
 <br />
 
-## A Go Implementation of the Polkadot Host
-
 > **Warning**
-> 2022-11-01: Gossamer is pre-production software
+>
+> The Gossamer Polkadot Host is pre-production software [2022-12-01]
 
-Gossamer is an implementation of the [Polkadot Host](https://wiki.polkadot.network/docs/learn-polkadot-host): an execution environment for the Polkadot runtime, which is materialized as a Web Assembly (Wasm) blob.  In addition to running an embedded Wasm executor, a Polkadot Host must orchestrate a number of interrelated services, such as [networking](dot/network/README.md), block production, block finalization, a JSON-RPC server, [and more](cmd/gossamer/README.md).
+Gossamer is a [Golang](https://go.dev/) implementation of the
+[Polkadot Host](https://wiki.polkadot.network/docs/learn-polkadot-host): an
+execution environment for the Polkadot runtime, which is materialized as a Web
+Assembly (Wasm) blob. In addition to running an embedded Wasm executor, a
+Polkadot Host must orchestrate a number of interrelated services, such as
+[networking](dot/network/README.md), block production, block finalization, a
+JSON-RPC server, [and more](cmd/gossamer/README.md#client-components).
 
-For more information about Gossamer, check out the [Gossamer Docs](https://ChainSafe.github.io/gossamer).
+## Getting Started
 
-## Get Started
+To get started with Gossamer, follow the steps below to build the source code
+and start a development network.
 
 ### Prerequisites
 
-Install Go version [`>=1.18`](https://go.dev/dl/#go1.18)
+[Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) is required
+to acquire the Gossamer source code, and
+[Make](https://tilburgsciencehub.com/building-blocks/configure-your-computer/automation-and-workflows/make/)
+is used to build it. Building Gossamer requires version 1.18 or greater of
```

Comment:
It doesn't work with Go 1.19, this is blocked by #2794 then #2788 then #2719 (each blocked sequentially). Although each just need reviews really, they should be good to go.

```suggestion
is used to build it. Building Gossamer requires version 1.18 of 
```